tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -47,12 +47,101 @@ def setUp(self):
             },
         )
 
-    # Remove this method once we don't need to support both approaches
-    def _generate_span_ids(self) -> list[str]:
+    def load_trace(self):
+        self.root_event = self.create_event(
+            trace_id=self.trace_id,
+            transaction="root",
+            spans=[
+                {
+                    "same_process_as_parent": True,
+                    "op": "http",
+                    "description": f"GET gen1-{i}",
+                    "span_id": root_span_id,
+                    "trace_id": self.trace_id,
+                }
+                for i, root_span_id in enumerate(self.root_span_ids)
+            ],
+            measurements={
+                "lcp": 1000,
+                "fcp": 750,
+                "fid": 3.5,
+            },
+            parent_span_id=None,
+            file_io_performance_issue=True,
+            slow_db_performance_issue=True,
```

Comment:
@wmak I brought back this change from your PR
<img width="584" alt="image" src="https://github.com/getsentry/sentry/assets/44410/8f459f11-4980-4b02-aaf5-f777d450a8ad">

Your change is the only change to `load_traces` since I merged:
<img width="440" alt="image" src="https://github.com/getsentry/sentry/assets/44410/ac26c561-95c6-4ac6-8b12-815c28762de6">